spirv_cross/src/wrapper.cpp

```diff
@@ -182,6 +182,11 @@ extern "C"
         INTERNAL_RESULT(((spirv_cross::Compiler *)compiler)->set_decoration(id, decoration, argument);)
     }
 
+    ScInternalResult sc_internal_compiler_set_name(const ScInternalCompilerBase *compiler, const uint32_t id, const char* name)
+    {
+        INTERNAL_RESULT(((spirv_cross::Compiler *)compiler)->set_name(id, std::string(name));)
```

Comment:
Do we rely on SPIRV-Cross to copy this string? i.e. what happens when `std::string(name)` goes out of scope

Reply:
We rely on SPIRV-Cross to copy the string, which it does.